client/src/components/Comparison/index.css

```diff
@@ -216,8 +226,8 @@
   display: flex;
   width: 79.9%;
   background-color: #E7E7E7;
-  height: 500px;
-  border-top-right-radius: 15px;
+  height: 600px;
+  /* border-top-right-radius: 15px; */
```

Comment:
@SalwaMugh You could just delete these commented lines unless you're considering re-adding them.